vtex-apps/app-store
# Remove Coming Soon shelf from homepage

#### What is the purpose of this pull request?

Remove the `Coming Soon` shelf from the Homepage

#### What problem is this solving?

The `Coming Soon` shelf does not have any business need now

#### How should this be manually tested?

Visit [this linked homepage](https://artur--extensions.myvtex.com/) to check it out

#### Screenshots or example usage

None

#### Types of changes
